lib/_tls_wrap.js

```diff
@@ -1056,18 +1056,14 @@ exports.connect = function(...args /* [port,] [host,] [options,] [cb] */) {
     socket.once('secureConnect', cb);
 
   if (!options.socket) {
-    var connect_opt;
-    if (options.path && !options.port) {
-      connect_opt = { path: options.path };
-    } else {
-      connect_opt = {
-        port: options.port,
-        host: options.host,
-        family: options.family,
-        localAddress: options.localAddress,
-        lookup: options.lookup
-      };
-    }
+    const connect_opt = {
+      path: options.path,
```

Comment:
I would prefer a documentation change, which explicitly states that if `path` is present, that gets first preference.

Reply:
That's already stated in the `tls` doc. I'll add it to `net`.